### PR TITLE
CORE-39905 remove config overrides

### PR DESCRIPTION
**Pull request by jonsnyder**
## Description

Removing client config overrides.  You can only set these on the server.

urlDestinationsEnabled
storedDestinationsEnabled
idSyncEnabled
idSyncContainerId
datasetId
schemaId

## Related Issue

https://jira.corp.adobe.com/browse/CORE-39905

## Motivation and Context

Trying to simplify the library and confusion on where to set configs.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [x] I have submitted a [documentation](https://github.com/AdobeDocs/alloy-docs) pull request or no changes are needed.
- [x] I have made any necessary test changes and all tests pass.
- [x] I have run the Sandbox successfully.

### CODE REVIEW

File: test/functional/sandbox/html/bogusCommand.html [hunk 1]
```diff
@@ -20,8 +20,7 @@
         configId: "9999999",
         orgId: "53A16ACB5CC1D3760A495C99@AdobeOrg",
         debugEnabled: true,
-        prehidingStyle: "section:nth-of-type(1) { opacity: 0 !important }",
-        idSyncContainerId: 81
+        prehidingStyle: "section:nth-of-type(1) { opacity: 0 !important }"
```
Comment (liljenback) [inline test/functional/sandbox/html/bogusCommand.html]: We have the configId 9999999 defined to get the idSyncContainerId 81 back?

**Comment by liljenback** [comment]
Will the configuration service know to provide old configuration parameters if an old library requires deprecated ones?

File: test/functional/sandbox/html/bogusCommand.html (hunk 1 above)
Comment (jonsnyder) [reply test/functional/sandbox/html/bogusCommand.html]: Yes that should be configured on the server.  But I don't think Alloy cares what the idSyncContainerId is at all.

**Comment by jonsnyder** [comment]
> Will the configuration service know to provide old configuration parameters if an old library requires deprecated ones?

My understanding is that Konductor doesn't send configuration to Alloy, but Conductor behaves differently based on the server config.  Alloy didn't really care about the configuration parameters that I removed except to be able to send them to Konductor.

**Pull request merged**